ArduPlane/ArduPlane.cpp

```diff
@@ -968,6 +969,77 @@ void Plane::update_flight_stage(void)
     airspeed.set_EAS2TAS(barometer.get_EAS2TAS());
 }
 
+/*
+*  ArduSoar support function
+*
+*  Peter Braswell and Samuel Tabor
+*/
+void Plane::update_soaring() {
+    
```

Comment:
I'd prefer update_soaring() to be in ArduPlane/soaring.cpp, along with any other soaring related code